Temporary workaround to prevent blank `about:support` template.

It seems querying the environment causes the thread to hang/exit.

This is related to the historical `firefox-02-environ.patch`, which no
longer seems to work as expected.

Finessing the firefox-02-environ.patch *should* clear the need for this patch.


--- firefox-115.0/toolkit/modules/Troubleshoot.sys.mjs-	2023-07-03 13:12:21.623735452 +0000
+++ firefox-115.0/toolkit/modules/Troubleshoot.sys.mjs	2023-07-05 19:22:21.911879789 +0000
@@ -451,7 +451,7 @@
       })
     );
   },
-
+/*
   async environmentVariables(done) {
     let Subprocess;
     try {
@@ -476,7 +476,7 @@
     }
     done(filteredEnvironment);
   },
-
+*/
   modifiedPreferences: function modifiedPreferences(done) {
     done(getPrefList(name => Services.prefs.prefHasUserValue(name)));
   },
